src/core/sagas/categories.js

```diff
@@ -1,16 +1,13 @@
-// Disabled because of
-// https://github.com/benmosher/eslint-plugin-import/issues/793
-/* eslint-disable import/order */
```

Comment:
are we missing `/* @flow */` here?

Reply:
this file will be converted to flow in #6935